shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -720,15 +773,26 @@ public void onMessage(Object message, BasicMessageChannel.Reply<Object> reply) {
                 break;
             }
             case "tooltip": {
-                AccessibilityEvent e = obtainAccessibilityEvent(ROOT_NODE_ID, AccessibilityEvent.TYPE_WINDOW_STATE_CHANGED);
+                AccessibilityEvent e = obtainAccessibilityEvent(
+                        ROOT_NODE_ID, AccessibilityEvent.TYPE_WINDOW_STATE_CHANGED);
                 e.getText().add((String) data.get("message"));
                 sendAccessibilityEvent(e);
             }
+            // Requires that the node id provided corresponds to a live region, or TalkBack will
+            // ignore the event.
```

Comment:
Add: "The event will cause talkback to read out the new label even if node is not focused" or something like that?

Reply:
Done